### ISSUE TITLE
Add default category when a new game is added.

### ISSUE THREAD

**Issue by vinguan** [JosefNemec/Playnite#1098]
Example: New game imported to library automatically associated with "Unplayed" category".

**Comment by despiojo** [comment]
I think with #1096 commit you will be able to obtain something similar, don't you?

**Comment by vinguan** [comment]
@despiojo No, what i what is when a new game is added to the library one of my custom categories of choice will be automatically added, like the example i did.
